api/src/main/java/org/openmrs/module/adminui/utils/RegexUtil.java

```diff
@@ -0,0 +1,60 @@
+package org.openmrs.module.adminui.utils;
+
+/**
+ * Evaluates a string for a given pattern
+ */
+public class RegexUtil {
+
+	/**
+	 * TODO: Confirm the correct pattern for the following regex.
+	 */
+	public static final String MUST_CONTAIN_DIGIT_REGEX = "(?=.*\\d).+";
+	public static final String MUST_CONTAIN_NONE_DIGIT_REGEX = "(?=.*[a-z])(?=.*[A-Z])(?=.*\\d).+";//"(?!^[0-9]*$)(?!^[a-zA-Z]*$)^([a-zA-Z0-9])";
```

Comment:
This seem like it will won't allow a password like 222#45 yet the '#' is a non digit